pmahnke/design.ubuntu.com
# Implement new font interaction

## Done
- Created a `design-p-font-tester` pattern
- Upgraded `vanilla-brochure-theme` to `1.0.4`
- Build the markup for the font tester
- Created a CSS only interaction
- Remove font-js from the codebase

## QA
- Pull down this branch
- Run:
```bash
bundle install
rm -rf node_modules
npm i
bundle exec jekyll serve
```
- Go to http://127.0.0.1:4000/font/
- Check that the font interaction works
- Check on small screens

## Details
Card: https://trello.com/c/s4U9fHLV
